plugins/labhub.py

```diff
@@ -222,6 +222,18 @@ def create_issue_cmd(self, msg, match):
                 target=user,
             )
 
+    @staticmethod
+    def profile_link(self, msg):
```

Comment:
Why does a `staticmethod` needs `self` object?

Reply:
Oh I might have committed the old file :(
Fixed